.github/workflows/pre-commit.yml

```diff
@@ -11,3 +11,11 @@ jobs:
     - uses: pre-commit/action@v3.0.0
       with:
         extra_args: "detect-secrets --all-files"
```

Comment:
post-merge-actions-build.yml and post-merge-actions-dev.yml build container images. Is it good to get this values in pre-commit ?